### ISSUE TITLE
windows support

### ISSUE THREAD

**Issue by gleba** [reicheltp/dgraph-client#1]
Hi!
i think we can rewrite `update-protos.sh` to node.js script
```
npm ERR! dgraph-client@0.1.15 install: `./update-protos.sh`
npm ERR! Exit status 1
```

[npm-debug.txt](https://github.com/reicheltp/dgraph-client/files/990454/npm-debug.txt)

**Comment by reicheltp** [comment]
@gleba Good idea! A PR is welcome. Otherwise, it'll take some time until I get this done.
